### PR TITLE
Add link to TF module tutorial and example TF modules

### PR DESCRIPTION
**Pull request by BeenzSyed**
* Add link to TF tutorial
* Add links to print TF module and template for TF modules

### CODE REVIEW

File: website/content/docs/integrate/nia-integration.mdx [hunk 1]
```diff
@@ -44,8 +44,8 @@ Consul-Terraform-Sync compatible Terraform module development process is fairly
 
 - Consul [documentation](/docs)
 - Consul-Terraform-Sync [documentation](/docs/nia)
-- Writing Consul-Terraform-Sync compatible Terraform modules [guide](/docs/nia/terraform-modules)
-- Example [module](https://registry.terraform.io/modules/PaloAltoNetworks/dag-nia/panos/latest) for reference
+- Writing Consul-Terraform-Sync compatible Terraform modules using our [guide](/docs/nia/terraform-modules) and [tutorial](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-module?in=consul/network-infrastructure-automation)
+- Example Terraform Modules for reference: [Panos](https://registry.terraform.io/modules/PaloAltoNetworks/dag-nia/panos/latest), [Simple Print Module](https://registry.terraform.io/modules/findkim/print/cts/latest) and a [Template to structure your Terraform Modules](https://github.com/hashicorp/consul-terraform-sync-template-module)
```
Comment (findkim) [inline website/content/docs/integrate/nia-integration.mdx:48]:
```suggestion
- Example Terraform Modules for reference: [PAN-OS](https://registry.terraform.io/modules/PaloAltoNetworks/dag-nia/panos/latest), [Simple Print Module](https://registry.terraform.io/modules/findkim/print/cts/latest) and a [Template to structure your Terraform Modules](https://github.com/hashicorp/consul-terraform-sync-template-module)
```

I think it's `PAN-OS`

Comment (findkim) [reply website/content/docs/integrate/nia-integration.mdx:48]: https://docs.paloaltonetworks.com/pan-os.html

i've also seen [`panos provider`](https://registry.terraform.io/providers/PaloAltoNetworks/panos/latest/docs) all lower case but I don't know if it applies here to the module

Comment (BeenzSyed) [reply website/content/docs/integrate/nia-integration.mdx:48]: I'll use PAN-OS. Thank you!

**Review by findkim — APPROVED** [review]
✨

**Pull request merged**